[en] update assign-pod-node.md (#31415)
kubernetes/website: content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

        image: redis:3.2-alpine
```

> The below yaml snippet of the webserver deployment has `podAntiAffinity` and `podAffinity` configured. This informs the scheduler that all its replicas are to be co-located with pods that have selector label `app=store`. This will also ensure that there will not be two web-server replicas co-located on a single node.

Reviewer: there will not be two web-server replicas --> there will be no more than one web-server replica ?

> The below yaml snippet of the webserver deployment has `podAntiAffinity` and `podAffinity` configured. This informs the scheduler that all its replicas are to be co-located with pods that have selector label `app=store`. This will also ensure that there will not be two web-server replicas co-located on a single node.

Reviewer 2: +1 to @tengqm suggestion above. Phrased it as a suggestion.

```suggestion
The below yaml snippet of the webserver deployment has `podAntiAffinity` and `podAffinity` configured. This informs the scheduler that all its replicas are to be co-located with pods that have selector label `app=store`. This will also ensure that there won't be more than one web-server replica co-located on a single node.
```

> The below yaml snippet of the webserver deployment has `podAntiAffinity` and `podAffinity` configured. This informs the scheduler that all its replicas are to be co-located with pods that have selector label `app=store`. This will also ensure that there won't be more than one web-server replica co-located on a single node.

Reviewer 3: ```suggestion
The below yaml snippet of the webserver deployment has `podAntiAffinity` and `podAffinity` configured. This informs the scheduler that all its replicas are to be co-located with pods that have selector label `app=store`. This ensures that there won't be more than one web-server replica co-located on a single node.
```
